api/core/v1alpha1/device_types.go

```diff
@@ -16,7 +20,7 @@ type DeviceSpec struct {
 	// Bootstrap is an optional configuration for the device bootstrap process.
```

Comment:
```suggestion
	// Provisioning is an optional configuration for the device provisioning process.
```
nit: this comment should be updated.